figmagic.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This change seems unrelated. IMHO, we should remove figmatic from `@bjerkio/brand`.